modules/distribution/src/repository/resources/conf/default.json

```diff
@@ -190,5 +190,67 @@
 
 
   "identity.auth_framework.endpoint.tenant_list_enabled" : false,
-  "identity.auth_framework.endpoint.mutual_ssl_manager_enabled" : true
+  "identity.auth_framework.endpoint.mutual_ssl_manager_enabled" : true,
+
+
+  "identity.entitlement.policy_point": {
+    "pdp": {
+      "enabled": false,
+      "default_caching_interval": "300",
+      "caching": {
+        "decision_caching": {
+          "enabled": true,
+          "caching_interval": "{{identity.entitlement.policy_point.pdp.default_caching_interval}}"
```

Comment:
This should be "$conf{identity.entitlement.policy_point.pdp.default_caching_interval}"

Reply:
Fixed by [70dd62b](https://github.com/wso2/product-is/pull/4513/commits/70dd62bff4e55f41924bfdb5daf427f11694d2f6)